controller-server/src/main/java/com/yahoo/vespa/hosted/controller/proxy/ConfigServerRestExecutorImpl.java

```diff
@@ -89,7 +89,7 @@ public ProxyResponse handle(ProxyRequest proxyRequest) throws ProxyException {
                 .orElseGet(() -> new ArrayList<>(zoneRegistry.getConfigServerUris(zoneId)));
 
         StringBuilder errorBuilder = new StringBuilder();
-        if (queueFirstServerIfDown(allServers, proxyRequest)) {
+        if (allServers.size() > 1 && queueFirstServerIfDown(allServers, proxyRequest)) {
```

Comment:
This was already checked in `queueFirstServerIfDown` ;0

Reply:
Nice, removed.